config/mapping.json

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
A future refactoring could add `ember-modules-codemod` as a dependency, and then require the JSON file from there. Basically, the goal being to only have this mapping have a single canonical location...